java-client/src/main/java/org/opensearch/client/opensearch/indices/stats/ShardStats.java

```diff
@@ -33,34 +33,15 @@
 package org.opensearch.client.opensearch.indices.stats;
 
 import jakarta.json.stream.JsonGenerator;
-import java.util.function.Function;
-import javax.annotation.Nullable;
-import org.opensearch.client.json.JsonpDeserializable;
-import org.opensearch.client.json.JsonpDeserializer;
-import org.opensearch.client.json.JsonpMapper;
-import org.opensearch.client.json.ObjectBuilderDeserializer;
-import org.opensearch.client.json.ObjectDeserializer;
-import org.opensearch.client.json.PlainJsonSerializable;
-import org.opensearch.client.opensearch._types.BulkStats;
-import org.opensearch.client.opensearch._types.CompletionStats;
-import org.opensearch.client.opensearch._types.DocStats;
-import org.opensearch.client.opensearch._types.FielddataStats;
-import org.opensearch.client.opensearch._types.FlushStats;
-import org.opensearch.client.opensearch._types.GetStats;
-import org.opensearch.client.opensearch._types.IndexingStats;
-import org.opensearch.client.opensearch._types.MergesStats;
-import org.opensearch.client.opensearch._types.RecoveryStats;
-import org.opensearch.client.opensearch._types.RefreshStats;
-import org.opensearch.client.opensearch._types.RequestCacheStats;
-import org.opensearch.client.opensearch._types.SearchStats;
-import org.opensearch.client.opensearch._types.SegmentsStats;
-import org.opensearch.client.opensearch._types.StoreStats;
-import org.opensearch.client.opensearch._types.TranslogStats;
-import org.opensearch.client.opensearch._types.WarmerStats;
+import org.opensearch.client.json.*;
```

Comment:
⛺️: Simplification and rearranging of imports. Happy to exclude and leave imports as is for this change. 

Let me know if I should revert.

Reply:
Yes, this causes a spotless check violation and fails CI, see https://github.com/opensearch-project/opensearch-java/actions/runs/10668564423/job/29568601968?pr=1177.

Reply:
Understood! Done.